opts/opts.go

```diff
@@ -22,6 +22,7 @@ var (
 	// is not supplied. A better longer term solution would be to use a named
 	// pipe as the default on the Windows daemon.
 	DefaultHTTPPort = 2375 // Default HTTP Port
+	DefaultTLSHTTPPort = 2376 // Default TLS encrypted HTTP Port
```

Comment:
yeah, I feel a bit weird always linking to it, but I'll do that when I gofmt -I've just replaced my notebook OS with win-server 2016 <:} so it'll be yet more downloading